avionics/common/src/avionics.cpp

```diff
@@ -130,27 +128,17 @@ int main(void) {
     SerialUSB.println("Initializing...");
 #endif
 
-    /* init sensors*/
-    SensorCollectionPtr sensors_ptr = getSensors();
-    SensorCollection &sensors = *sensors_ptr;
-
-    /* init ignitors*/
-    IgnitorCollectionPtr ignitors_ptr = getIgnitors();
-    IgnitorCollection &ignitors = *ignitors_ptr;
-
-    /* init log file */
-    constexpr char LOG_FILE_NAME[] = "datalog.csv";
-    CSVWrite<CSVWriteImpl> datalog(LOG_FILE_NAME);
-
-    RocketStatus statusOfInit = collectStatus(sensors, ignitors);
-    displayStatus(statusOfInit, buzzer);
-
-    RadioController radio{Hal::SerialInst::Radio};
-
-    Calculator calc(sensors);
-
-    StateMachine state_machine;
-    if (statusOfInit == RocketStatus::CRITICAL_FAILURE) {
+    Rocket rocket;
+    // Some aliases, to reduce amount of code changed at once
+    auto &state_machine = rocket.state_machine;
+    auto &init_status = rocket.init_status;
+    auto &radio = rocket.radio;
+    auto &sensors = rocket.sensors;
+    auto &ignitors = rocket.ignitors;
+    auto &calc = rocket.calc;
+    auto &datalog = rocket.datalog;
```

Comment:
Might be less confusing for new readers to not do this and let the code be more explicit instead.

Reply:
Main purpose of the rocket object is to serve as a container for all of these different variables - practically every variable in the `main` function is kept under the `Rocket` object. So I'm not sure how more readable it would be to just prepend every variable with `rocket` - if it's everywhere, I'd think it becomes noise